internal/svc/admin_svc.go

```diff
@@ -104,6 +105,52 @@ func (cas *CerbosAdminService) AddOrUpdateSchema(ctx context.Context, req *reque
 	return &responsev1.AddOrUpdateSchemaResponse{}, nil
 }
 
+func (cas *CerbosAdminService) ListPolicies(ctx context.Context, req *requestv1.ListPoliciesRequest) (*responsev1.ListPoliciesResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	policyIds, err := cas.store.ListPolicyIDs(context.Background())
+	if err != nil {
+		ctxzap.Extract(ctx).Error("Could not get policy ids", zap.Error(err))
+		return nil, status.Error(codes.Internal, "could not get policy ids")
+	}
+
+	sortPolicies(req.SortOptions, policyIds)
+	return &responsev1.ListPoliciesResponse{
+		PolicyIds: policyIds,
+	}, nil
+}
+
+func (cas *CerbosAdminService) GetPolicy(ctx context.Context, req *requestv1.GetPolicyRequest) (*responsev1.GetPolicyResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	log := ctxzap.Extract(ctx)
+	policies := make([]*policyv1.Policy, 0, len(req.Id))
+	for _, id := range req.Id {
+		p, err := cas.store.LoadPolicy(ctx, id)
+		if err != nil {
+			log.Error(fmt.Sprintf("Could not get the policy with id %s", id), zap.Error(err))
+			return nil, status.Errorf(codes.Internal, "could not get the policy with id %s", id)
+		}
+		policies = append(policies, p.Policy)
+	}
```

Comment:
When you know the size of the slice in advance, it's slightly faster to use indexing rather than `append`. I would rewrite it as follows:

```suggestion
	policies := make([]*policyv1.Policy, len(req.Id))
	for i, id := range req.Id {
		p, err := cas.store.LoadPolicy(ctx, id)
		if err != nil {
			log.Error(fmt.Sprintf("Could not get the policy with id %s", id), zap.Error(err))
			return nil, status.Errorf(codes.Internal, "could not get the policy with id %s", id)
		}
		policies[i] =  p.Policy
	}
```

There are some other places above where this can be done as well.

```
BenchmarkArrayAppend
BenchmarkArrayAppend/append
BenchmarkArrayAppend/append-8         	23759680	       43.08 ns/op	     23 B/op	      0 allocs/op
BenchmarkArrayAppend/index
BenchmarkArrayAppend/index-8          	32172147	       41.67 ns/op	     23 B/op	      0 allocs/op
```